SWLOR.Game.Server/Entity/Creature.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace SWLOR.Game.Server.Entity
+{
+    public class Creature : EntityBase
```

Comment:
Let's call this a DMCreature or something to that effect. Creature is pretty ambiguous

Reply:
Done